marine/marine_pool.py

```diff
@@ -10,21 +10,20 @@ class MarinePool:
     _marine_instance: ClassVar[Optional[Marine]] = None
 
     def __init__(
-        self, lib_path: str, epan_auto_reset_count: int, process_count: int = 4
+        self,
+        lib_path: str,
+        epan_auto_reset_count: int,
+        process_count: int = 4,
+        preferences: Optional[List[str]] = None,
     ):
+        self.pool = None
         self._lib_path = lib_path
         self._epan_auto_reset_count = epan_auto_reset_count
         self._process_count = process_count
+        self.preferences = preferences
```

Comment:
Why is this not private like the others?

Reply:
why should it? its a nice way to get the perefenrces here

Reply:
What's the application of getting the preferences? I already know them since I passed them to the constructor